packages/medusa-file-local/.npmignore

```diff
@@ -0,0 +1,8 @@
+.DS_store
```

Comment:
todo: This file is not necessary as it will be published why is specified in the package.json `files`